README.md

```diff
@@ -40,26 +50,67 @@ We plan to support the following key features of LoRaWAN 1.0.2 specification, ho
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+
+
+The ARM template will deploy all the required Azure infrastructure to get you started quickly. Specifically, it configures the required Azure Services and  Azure IoT Edge modules. As soon as you connect your IoT Edge device these modules will be pushed onto the gateway and you should have an Azure connected LoRaWAN gateway running. 
+
+- The template will deploy the following resources in your Azure subscription:
+  - [IoT Hub](https://azure.microsoft.com/en-us/services/iot-hub/)
+  - [Azure Functions](https://azure.microsoft.com/en-us/services/functions/)
+  - [Redis Cache](https://azure.microsoft.com/en-us/services/cache/)
+
+- The LoRaWAN gateway will be provisioned with two Azure IoT Edge modules each for the [packet forwarder](https://github.com/Lora-net/packet_forwarder) and a network server.
 
-We have a variety of ways you can  get started with the kit, chose the appropriate documentation based on your persona and applicability.
-- **Setup a LoRaWAN Gateway**: We provide an easy to use Azure ARM template and deployment guidance to get you quickly started with the LoRaWAN starter kit. Use the [Quick Start](/Docs/quickstart.md) to setup a LoRaWAN Gateway and connect to LoRA end nodes. 
-- **Upgrade an existing installation**: Refer to the [upgrade guide](/Docs/upgrade.md) for instructions and tips for a clean upgrade.
-- **Develop and debug the LoRaWAN starter kit**: If you are a developer and want to contribute or customize the LoRaWAN starter kit, refer to our [Developer Guidance](/Docs/devguide.md) for more details on how to build, test and deploy the kit in your dev environment. We also support a [simulator](/Docs/simulator.md) that allows for developing without the need of an actual device gateway.
-- **Enable a gateway or device to be compatible with the starter kit**: We have developed the LoRaWAN starter kit agnostic of a device manufacturer implementation and focussed on the specifics on underlying architectures (arm, x86). However, we understand that device manufacturers can have specific requirements; these could be specific to a gateway and the packet forwarders they use or to the LoRa nodes and the decoders the device may use. We have provided specific instructions on making these specialized hardware compatible with our kit. You can follow these [instructions](/Docs/partner.md) depending on your scenarios and also have your device gateway highlighted on our repo.
+#### Configuration for the ARM template  
+- Make sure to provide your gateway's reset pin in the dialog before the deployment.
+- if your gateway use SPI_DEV version 1 the packet forwarder module will not work out-of-the-box. To fix this, simply add an environment variable 'SPI_DEV' set to the value 1 to the LoRaWanPktFwdModule module (SPI_DEV is set to 2 by default).
```

Comment:
"If" should be capitalized. Also add "," after the wrods version 1 --> If your gateway use SPI_DEV version 1, the packet forwarder module will not work out-of-the-box.